src/index.ts

```diff
@@ -1,10 +1,12 @@
 import ejs from 'ejs'
-import { helpers, getContext, Property, Parameter } from "@dylibso/xtp-bindgen"
+import { helpers, getContext, Property, Parameter, } from "@dylibso/xtp-bindgen"
 
 function toTypeScriptType(property: Property | Parameter): string {
   let tp
   if (property.$ref) {
     tp = property.$ref.name
+  } else if (property.additionalProperties) {
+    tp = `Record<string, ${toTypeScriptType(property.additionalProperties)}>`
```

Comment:
Why use Record over Map? my understanding is Record is pretty limited to the key and value types it can hold.

Reply:
I think i see the logic here. looks like keys can only really be strings anyway.

Reply:
I don't have a strong opinion here, but it seems like `Record` has a more straightforward path for json serialization: https://stackoverflow.com/questions/50153172/how-to-serialize-a-map-in-javascript